templates/tfengine/components/devops/main.tf

```diff
@@ -104,13 +104,25 @@ module "owners_group" {
     module.project
   ]
 }
+
+resource "time_sleep" "owners_wait" {
+  depends_on = [
+    module.owners_group,
+  ]
+  create_duration = "10s"
```

Comment:
Is 10s definitely more than enough? It should be enough that this is essentially never flaky.

Reply:
I tried 5s before and was a bit flaky so changed to 10s. Let me do 15s to be safe.